schemas/context/segmentdefinition-meta-base.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segment-meta-base",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segments basic metadata",
```

Comment:
Is this metadata or the required fields for a segment? This is the first time I've seen a schema definition described as metadata.